src/connection.rs

```diff
@@ -153,11 +153,20 @@ impl IntoConnectionInfo for url::Url {
     }
 }
 
+struct TcpConnection {
+    reader: BufReader<TcpStream>,
+    open: bool
+}
+
+struct UnixConnection {
```

Comment:
Yikes, care to submit a PR with that change?

Reply:
@repi Sorry about that.

@badboy I noticed that the specs also do not work on windows, I'll send a PR to fix that tomorrow. Would you be interested in an appvoyor config to prevent broken windows builds in the future?

Reply:
Absolutely. Having this fully tested on Windows would be great.